.github/actions/create-package-for-el-7/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+# Inputs
+
+PE_VERSION=${1}
+PLATFORM=${PLATFORM:-el-7}
+GITHUB_WORKSPACE=${GITHUB_WORKSPACE:-/workspace}
+SCRATCHDIR=${SCRATCHDIR:-/scratch}
+
+if [ -z "${PE_VERSION}" ]; then
+  echo 'ERROR: Did not recieve a valid PE version argument ($1)'
```

Comment:
This won't expand with single quotes

In test.sh line 12:
  echo 'ERROR: Did not recieve a valid PE version argument ($1)'
       ^-- SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.